test/regress/run_regression

```diff
@@ -49,6 +49,16 @@ cvc4=$1
 benchmark_orig=$2
 benchmark="$benchmark_orig"
 
+if [ ! -z "$TEST_GROUP" ]; then
```

Comment:
If we go with this solution, we should document the effect these have at the top of the file.